src/Integration.Vsix/AsmRef_Integration.Vsix_Baseline_WithStrongNames.txt

```diff
@@ -298,13 +298,14 @@ Referenced assemblies:
 - 'PresentationCore, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'
 - 'PresentationFramework, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'
 - 'SonarLint.VisualStudio.ConnectedMode, Version=7.3.0.0, Culture=neutral, PublicKeyToken=c5b62af9de6d7244'
+- 'SonarLint.VisualStudio.Infrastructure.VS, Version=7.3.0.0, Culture=neutral, PublicKeyToken=c5b62af9de6d7244'
```

Comment:
@bigfluffycookie FYI the CI build was failing because there was a difference between the baseline version of the AsmRef file and the version generated on the CI machine during the build.

I checked out your branch locally and rebuilt, which automatically updated the baseline file to include this change. `SL.VS.TeamExplorer` now uses the service operation class from the `SL.VS.Infrastructure` assembly so it is expected that it will reference the `SL.VS.Infrastructure` assembly.